css/properties/align-items.json

```diff
@@ -8,13 +8,28 @@
             "webview_android": {
               "version_added": "4.4"
             },
-            "chrome": {
-              "version_added": "21",
-              "notes": "Older versions of the specification treat absolute positioned children as though they are a 0 by 0 flex item. Later specification versions take the children out of the flow and set their positions based on align and justify properties. Chrome implements the new behavior beginning with Chrome 52."
-            },
-            "chrome_android": {
-              "version_added": null
-            },
+            "chrome": [
```

Comment:
The first element of the `chrome` and `chrome_android` support array should be:
```json
{
  "version_added": "52"
}
```
followed by the current contents of this array.
This is because Chrome 52 now implements this fully.

Reply:
Fixed! Also added this for `justify-content` and `order` until @jpmedley has confirmed.

Reply:
Here's the full list of what's supported in Chrome: https://cs.chromium.org/chromium/src/third_party/blink/renderer/core/css/CSSValueKeywords.json5?l=520

Reply:
Sorry, I don't quite understand that to do with this.

Reply:
justify-content is supported on Chromium browsers as is anything else not commented out of that list. Were you not asking me to confirm the status of justify-content in Chrome?

Reply:
Oh, my bad. The comment is poorly formulated. And it concerned `align-self`, not `justify-content`. What I meant to say was that added the note about absolute positioned flex children for `align-self` too after reading [this](https://www.chromestatus.com/feature/6600926009753600):
> However, the latest version of the spec takes them fully out of flow and sets the static position based on **align and justify properties**.

And I was wondering if that would be correct? Does that changed behavior affect the `order` property as well?

Reply:
It doesn't appear so.

Sorry about that link. It was supposed to link to align-self. Shortly after I posed it, a new version of the source file got merged to trunk which made the line number wrong.

Reply:
Ok, thanks :+1: